src/EditorFeatures/Core.Cocoa/Microsoft.CodeAnalysis.EditorFeatures.Cocoa.csproj

```diff
@@ -30,4 +30,8 @@
     <PackageReference Include="XamarinMac" Version="$(XamarinMacVersion)" />
     <PackageReference Include="System.ValueTuple" Version="$(SystemValueTupleVersion)" />
   </ItemGroup>
+  <ItemGroup>
+    <PublicAPI Include="PublicAPI.Shipped.txt" />
+    <PublicAPI Include="PublicAPI.Unshipped.txt" />
```

Comment:
💭  I thought this was done in Directory.Build.props

Reply:
Other projects list them explicitly in the project file.